Add ERD and database with test data

yash-puligundla opened the pull request:
The following changes have been added as a part of this PR
- ERD
- database creation and population scripts for postgresql and sqlite3
- Makefile with database creation, population commands for postgres and sqlite3

jb-adams reviewed (commented):
Looks great!

Here are some of my ideas:

- Can we add a `platform` table upstream of `report_series`, a platform means a web service or series of services (e.g. ENA's refget service). We can connect platform to report series via foreign key, essentially saying "this report series is for this platform." One platform may have many report series, but each report series belongs to only one platform (one-to-many)

- Can we add an `organization` table upstream of `platform`. Organization being the implementing or hosting org (e.g. ENA, ELIXIR, DNAstack, GA4GH). One organization can have many platforms

- Can we add a `testbed` table upstream of `report_series`. Testbed means the actual application that is a valid GA4GH testbed (e.g. refget compliance, RNAget compliance, DRS + Passport). A `testbed_id` in `report_series` foreign key indicates what testbed the  series is associated with. 

- Delete `testbed_name`, `testbed_version`, and `testbed_description` from `report`, since that info can be gained from data joins to `testbed`

- Delete `platform_name` and `platform_description` from `report_series`, as that can be gained from data joins to `platform`

yash-puligundla commented:
- Updated PostgreSQL scripts to reflect the new ERD changes.
- changed the table name from "user" to "github_user" as user is a reserved keyword in PostgreSQL.

jb-adams commented on database/postgresql/create-tables.sql:
```diff
@@ -0,0 +1,190 @@
+CREATE TABLE IF NOT EXISTS ga4gh_testbed_summary
```
rename table to `summary`

jb-adams commented on database/ERD/ga4gh-testbed-erd.drawio:
```diff
@@ -0,0 +1 @@
+<mxfile host="app.diagrams.net" modified="2022-02-11T20:37:50.690Z" agent="5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.80 Safari/537.36" etag="e-LMuayBQHAK7EPHULVs" version="16.5.6" type="device"><diagram id="R2lEEEUBdFMjLlhIrx00" name="Page-1">7V1dl9q4lv01rDXzUFmWwRgeu5JUd26Svpl0587t+8JywICnwKaNK1WVXz82+AMsYSQsGaFzstIdcBnh0tnaRzqfvf7b9cuvsbdZfo ...
```
Can you rename table `user` to `github_user` in the ERD?

jb-adams reviewed (changes requested):
Looks great @yash-puligundla ! I think it's just about there, just a couple edits

yash-puligundla commented:
@jb-adams  Good catch. I have removed the duplicate summary table and changed "user" to "github_user" on the ERD

jb-adams reviewed (approved):
LGTM!
